test/services/GuidanceServiceSpec.scala

```diff
@@ -411,12 +410,31 @@ class GuidanceServiceSpec extends BaseSpec  with GuiceOneAppPerSuite {
       }
 
     }
+  }
+
+  "Calling getProcessContext full" should {
+
+    // "return a Process context containing legalPageIds" in new Test {
```

Comment:
Should this test be removed or corrected?

Reply:
beginnings of further tests